File: PHP/Algorithms/String/swap.php
<?php

/**
 * Swaps the characters of a given string
 *
 * @param string $string
 * @param int $i
 * @param int $j
 */
function swap(&$string, $i, $j)
{
    $temp = $string[$i];
    $string[$i] = $string[$j];
    $string[$j] = $temp;
}

File: C++/Algorithms/String/swap.cpp
/**
 * Swaps the characters of a given string
 *
 * @param char *string
 * @param int i
 * @param int j
 */
void swap(char string[], int i, int j)
{
    char temp = string[i];
    string[i] = string[j];
    string[j] = temp;
}